.github/workflows/handle.yml

```diff
@@ -68,8 +68,15 @@ jobs:
           token_format: 'access_token'
       - name: 'Install AOD CLI'
         run: 'go install github.com/abcxyz/access-on-demand/cmd/aod@${{ inputs.aod_cli_version }}'
-      # Duration labels need to be predixed with "duration-", an example is "duration-2h",
+
+      # Start handling IAM request.
+      - id: 'changed-file-iam'
+        uses: 'tj-actions/changed-files@84ed30e2f4daf616144de7e0c1db59d5b33025e3' # ratchet:tj-actions/changed-files@v35
+        with:
+          files: 'iam.yaml'
```

Comment:
Another way is to use hashFiles('iam.yaml') != '' to check the file existence. changed-files is more efficient since it won't handle the file request if no changes to the file, but it requires the file must exists in the triggering branch. Given the file will be created when setting up the repos, it makes sense to use changed-files here.

Reply:
Why can't we use a constraint on the job to only run when the file is changed? 

```yaml
on:
  push:
    paths:
      - 'iam.yaml'
  pull_request:
    paths:
      - 'iam.yaml'
```

Reply:
It is because we cannot use path constraint with pull_request_review event. Handle is triggered on this event when review status is approved, [here](https://github.com/abcxyz/aod-template/pull/1/files#diff-898d9cdbbc9e945afcaf482b25e6e81297319146b3887a322faf9368dbb56eabR5) is what the caller workflow will look like.

Reply:
Ah okay - can we document that really clearly? That wasn't immediately obvious from the YAML or the PR description.

Reply:
Sure, will add more comment!

Reply:
Does this even work? If `iam.yaml` doesn't exist, does this step fail? I think we should instead update the "Handle IAM Request" step to run:

```sh
if [ -f "iam.yaml" ]; then
  aod iam handle -path $FILE_PATH -duration $DURATION -start-time $START_TIME
else
  echo "::notice title=IAM Request::Skip because iam.yaml is not found"
fi
```

Reply:
Discussed offline, the iam.yaml file will only exist when a request is made, therefore we should not use this github action. switching...